moveit_planners/pilz_industrial_motion_planner/test/unit_tests/src/unittest_trajectory_functions.cpp

```diff
@@ -135,6 +135,35 @@ class TrajectoryFunctionsTestBase : public testing::Test
    */
   bool tfNear(const Eigen::Isometry3d& pose1, const Eigen::Isometry3d& pose2, double epsilon);
 
+  /**
+   * @brief check if two sets of joint positions are close
```

Comment:
Thanks for the docstring! Can you fill these in?

Reply:
Sure thing. I'm guessing the synrtax is just @param <name> <description> ?

Reply:
Done :+1: